app/helpers/application_helper.rb

```diff
@@ -1,12 +1,8 @@
 module ApplicationHelper
-  def flash_messages(opts = {})
-    flash.to_h.slice('notice', 'alert').each do |msg_type, message|
-      concat(content_tag(:div, message, class: "alert flash #{bootstrap_class_for(msg_type)} fade in") do
-        concat content_tag(:button, 'x', class: 'close', data: {dismiss: 'alert'})
-        concat safe_join(Array(message), tag(:br))
-      end)
-    end
-    nil
+  include ActiveModel
```

Comment:
Are you sure you want to do this? Including the whole of Active Model in the application helper?